snpdb/utils.py

```diff
@@ -40,6 +40,9 @@ def send(self):
         self.sent = True
         if slack_web_hook := empty_to_none(self.lab.slack_webhook):
             # send to lab's external slack
+            if settings.DEBUG:
+                self.blocks.insert(0, NotificationBuilder.HeaderBlock(f"Notification for {self.lab.name}"))
```

Comment:
To make this even more obvious, we start the header block with ":hospital:" which should put a hospital emoji at the start (otherwise all the bold text has a habit of running into each other)

Reply:
Note that is supposed to say
```
:hospital:
```
don't copy the emoji :)